src/resources/order/orders.controller.js

```diff
@@ -0,0 +1,38 @@
+import createOrder from './models/index.model';
+import { formatResponse } from '../../helpers/baseHelper';
+
+const create = async (req, res) => {
+  const { id } = req.user;
+  const {
+    shipping_address,
+    status,
```

Comment:
if status defaults to pending when a user creates order is there a need to add it here?